src/Components/common/getUrlInputFile.js

```diff
@@ -0,0 +1,12 @@
+
```

Comment:
the name of the file should be just image because with the current name you are limiting the use of this file just to the current function and its on a common file, so, all the common functionallities for a image should be here

Reply:
I do not understand what we refer to with just an image?